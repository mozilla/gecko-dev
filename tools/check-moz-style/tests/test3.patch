# Test
patch summary with no bug number

Some bogus patch description
diff --git a/tests/test2.cpp b/tests/test2.cpp
new file mode 100644
--- /dev/null
+++ b/tests/test2.cpp
@@ -0,0 +1,3 @@
+int main() {
+  return 0;
+}
