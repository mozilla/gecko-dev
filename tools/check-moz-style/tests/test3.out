patch:0:  Patch summary should begin with 'Bug XXXXX - ' or 'No bug -'.  [patch/bugnumber] [3]
tests/test2.cpp:1:  No copyright message found.  [legal/copyright] [3]
Done processing tests/test2.cpp
